assets/src/components/admin/admin_tables.tsx

```diff
@@ -575,14 +575,32 @@ const elevatorStatusColumn = {
   FormCell: FormTextarea,
 };
 
+const reconstructedAlertWidgetColumn = {
+  Header: "Alert Widget",
+  accessor: buildAppParamAccessor("alert_widget"),
+  mutator: buildAppParamMutator("alert_widget"),
```

Comment:
Is this to be a general column for anything alerts, or reconstructed specifically? If for reconstructed, should include it in the name since we already have an alert widget.